hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/codec/EntityNumCodec.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.file.impl.codec;
+
+import com.hedera.node.app.service.mono.utils.EntityNum;
+import com.hedera.pbj.runtime.Codec;
+import com.hedera.pbj.runtime.io.ReadableSequentialData;
+import com.hedera.pbj.runtime.io.WritableSequentialData;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import java.io.IOException;
+
+public class EntityNumCodec implements Codec<EntityNum> {
```

Comment:
Need javadoc on this public class and these public methods, to explain what they do.

Reply:
I just used the same class from mono service till we will extract it probably to spi